py-polars/polars/dataframe/frame.py

```diff
@@ -4509,7 +4510,9 @@ def limit(self, n: int = 5) -> Self:
         """
         return self.head(n)
 
-    def drop_nulls(self, subset: str | Collection[str] | None = None) -> DataFrame:
+    def drop_nulls(
+        self, subset: str | Collection[str] | SelectorType | None = None
```

Comment:
Same here, and in some other places. Maybe we should have a type alias for this combination?
```suggestion
        self, subset: str | Collection[str] | SelectorType | Collection[SelectorType] | None = None
```

Reply:
Yup, good plan; I'll commit the suggestions and then update with a TypeAlias 👍